src/app/routers/schema_router.py

```diff
@@ -74,9 +77,51 @@ async def get_schema_from_bucket(
         logger.error("Schema metadata not found")
         raise exceptions.ExceptionNoSchemaFound
 
-    logger.info("Bucket schema metadata location successfully retrieved.")
-    logger.debug(f"Bucket schema metadata location: {bucket_schema_filename}")
-    logger.info("Getting schema metadata...")
+    logger.info("Bucket schema location successfully retrieved.")
+    logger.debug(f"Bucket schema location: {bucket_schema_filename}")
+    logger.info("Getting schema...")
```

Comment:
Just housekeeping but it would be worth condensing the L80 and L82 Info logs into one info log, the L81 debug should still remain as is.

Reply:
Yeah, they should be combined. I will check whether I can merge some logs together